strr-web/components/bcros/status-card/StatusCard.vue

```diff
@@ -28,22 +28,25 @@
         :label="tRegistrationStatus('view')"
         class-name="px-4 py-1 mobile:grow-0"
         variant="ghost"
+        data-test-id="view-application-link"
```

Comment:
I think this would override the button's test id even without having it as a prop.

Reply:
Related to this: Please make sure that your date-test-id is unique. Right now I see many that are not. This can potentially give us issues in test automation. (Not a complete deal breaker but still)

Reply:
I don't think it does.
```
 FAIL  tests/unit/components/status-card/StatusCard.spec.ts > renders view application and download receipt links for paid applications
AssertionError: expected false to be true // Object.is equality

- Expected
+ Received

- true
+ false

 ❯ tests/unit/components/status-card/StatusCard.spec.ts:59:64
     57|   })
     58|
     59|   expect(wrapper.findTestId('view-application-link').exists()).toBe(true)
       |                                                                ^
     60|   expect(wrapper.findTestId('download-receipt-link').exists()).toBe(true)
     61| })
```